sdk/metricsadvisor/Azure.AI.MetricsAdvisor/api/Azure.AI.MetricsAdvisor.netstandard2.0.cs

```diff
@@ -1243,6 +1243,34 @@ public partial class MetricEnrichedSeriesData
         public static bool operator !=(Azure.AI.MetricsAdvisor.Models.MetricPeriodType left, Azure.AI.MetricsAdvisor.Models.MetricPeriodType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public static partial class MetricsAdvisorModelFactory
+    {
+        public static Azure.AI.MetricsAdvisor.Models.AnomalyAlert AnomalyAlert(string id = null, System.DateTimeOffset timestamp = default(System.DateTimeOffset), System.DateTimeOffset createdOn = default(System.DateTimeOffset), System.DateTimeOffset lastModified = default(System.DateTimeOffset)) { throw null; }
+        public static Azure.AI.MetricsAdvisor.Models.AnomalyAlertConfiguration AnomalyAlertConfiguration(string id = null, string name = null, string description = null, Azure.AI.MetricsAdvisor.Models.MetricAlertConfigurationsOperator? crossMetricsOperator = default(Azure.AI.MetricsAdvisor.Models.MetricAlertConfigurationsOperator?), System.Collections.Generic.IEnumerable<string> dimensionsToSplitAlert = null, System.Collections.Generic.IEnumerable<string> idsOfHooksToAlert = null, System.Collections.Generic.IEnumerable<Azure.AI.MetricsAdvisor.Models.MetricAlertConfiguration> metricAlertConfigurations = null) { throw null; }
+        public static Azure.AI.MetricsAdvisor.Models.AnomalyDetectionConfiguration AnomalyDetectionConfiguration(string id = null, string name = null, string description = null, string metricId = null, Azure.AI.MetricsAdvisor.Models.MetricWholeSeriesDetectionCondition wholeSeriesDetectionConditions = null, System.Collections.Generic.IEnumerable<Azure.AI.MetricsAdvisor.Models.MetricSeriesGroupDetectionCondition> seriesGroupDetectionConditions = null, System.Collections.Generic.IEnumerable<Azure.AI.MetricsAdvisor.Models.MetricSingleSeriesDetectionCondition> seriesDetectionConditions = null) { throw null; }
```

Comment:
although this is public, the reason you expose it is so people can set the `id`, right?

Reply:
Correct.